cats/templates/cats/congrats.html

```diff
@@ -0,0 +1,11 @@
+<head>
+    <title>Congrats!</title>
+</head>
+{% extends 'base.html' %}
+{% block content %}
+    <h1>Congrats!</h1>
+    <div style="text-align: center">
```

Comment:
```suggestion
    <div class="text-center">
```
and style:
```
.text-center {
    text-align: center;
}
```
This class looks like it can be used in other places so you can define `common.css`  or `global.css` file with styles like that and import it to your templates.

But I see that you linked bootstrap library in your base.html file, so just `text-center` class should be enough:
https://getbootstrap.com/docs/4.0/utilities/text/#text-alignment